pkg/pkgmgr/dpkg.go

```diff
@@ -83,7 +83,11 @@ func getAPTImageName(manifest *unversioned.UpdateManifest, osVersion string) str
 	osType := Debian
 
 	if manifest == nil || manifest.Metadata.OS.Type == Debian {
-		version = strings.Split(version, ".")[0] + "-slim"
+		if version > "11" {
```

Comment:
I added an additional check here because Ubuntu and kube-proxy images were failing if no vulnerability scan was passed in.

Images throwing the errors prior to this change:
`docker.io/library/ubuntu:jammy-20231004`
`us.gcr.io/k8s-artifacts-prod/kube-proxy:v1.27.1`